### ISSUE TITLE
NoMoreInteractions like in Mockito

### ISSUE THREAD

**Issue by dan-leech** [felangel/mocktail#13]
@felangel what do you think about NoMoreInteractions to be sure that no other methods were called?

**Comment by felangel** [comment]
Hi @dan-leech 👋 
Thanks for opening an issue!

Currently, you can use `verifyMocks` to ensure that all stubs were used like:

```dart
setUp(() {
  foo = MockFoo();
});

tearDown(() {
  verifyMocks(foo);
});
```

But I agree it would be nice to also introduce support for verifying no further interactions occurred 👍

**Comment by felangel** [comment]
This is available in v0.0.2-dev.2 🎉
